### PR TITLE
Color struct and class type keywords

### PR DESCRIPTION
**Pull request by darress**
I think the keyword int and struct type Int32 should be colored the same way as they are one and the same. This little change does just that.

### CODE REVIEW

File: SemanticColorizer/SemanticColorizer.cs [hunk 1]
```diff
@@ -100,6 +108,7 @@ class SemanticColorizer : ITagger<IClassificationTag>
               Cache doc,
               NormalizedSnapshotSpanCollection spans) {
             var snapshot = spans[0].Snapshot;
+            var comparer = StringComparer.InvariantCultureIgnoreCase;
```
Comment (arBmind) [inline SemanticColorizer/SemanticColorizer.cs:111]: It seems `comparer` is not used.

File: SemanticColorizer/SemanticColorizer.cs [hunk 2]
```diff
@@ -191,9 +203,7 @@ class SemanticColorizer : ITagger<IClassificationTag>
                   return Classifier.GetClassifiedSpans(model, textSpan, workspace);
               });
 
-            return from cs in classifiedSpans
-                   where comparer.Compare(cs.ClassificationType, "identifier") == 0
```
Comment (arBmind) [inline SemanticColorizer/SemanticColorizer.cs:195]: `comparer` is unused here as well.

File: SemanticColorizer/SemanticColorizer.cs [hunk 3]
```diff
@@ -8,6 +9,8 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel.Composition;
+using System.Diagnostics;
+using System.IO;
```
Comment (arBmind) [inline SemanticColorizer/SemanticColorizer.cs:13]: Are these namespaces really used?

File: SemanticColorizer/Constants.cs [hunk 4]
```diff
@@ -17,5 +17,7 @@ public static class Constants
         public const String LocalFormat = "s.semantic-colorizer.local";
         public const String TypeSpecialFormat = "s.semantic-colorizer.type-special";
         public const String TypeNormalFormat = "s.semantic-colorizer.type-normal";
+        public const String TypeClassFormat = "class name";
+        public const String TypeStructFormat = "struct name";
```
Comment (arBmind) [inline SemanticColorizer/Constants.cs:21]: these string values are default formats? - I guess there could be a named constant somewhere else in Roslyn.

**Review by arBmind — CHANGES_REQUESTED** [review]
It seems your changes are really C# specific. With this change Colorizer will not work for the other Roslyn languages.
I guess this would need a branch for this. The TypeSpecial & TypeNormal formats have to stay in use for all other languages.

I am also unsure if this is worth all the troubles.

**Comment by arBmind** [comment]
A similar feature was just merged.

Thank you for the inspiration!

**Comment by darress** [comment]
You are welcome.

**Pull request closed without merging**